pkgs/development/python-modules/requests/default.nix

```diff
@@ -54,7 +53,6 @@ buildPythonPackage rec {
     pytest-mock
     pytest-xdist
     pytestCheckHook
-    trustme
```

Comment:
`trustme` is/was needed for Python2 compatibility. I guess that we have to keep it as the package is not Python 3-only.

Reply:
Unfortunately, the python2 support is not there right now:
```
nix-build . -A python2Packages.requests
error: pytest-forked-1.4.0 not supported for interpreter python2.7
(use '--show-trace' to show detailed location information)
```